docs/INSTALL.K8S.md

```diff
@@ -110,6 +111,44 @@ nohup ./kubelet \
   2>&1 > /dev/null &
 ```
 
+Note that in latest kubernetes release (after 1.8.0), api-servers has been
+deprecated and you need to use kubeconfig to config api servers. Also,
+network-plugin-dir option has been changed to cni-conf-dir. e.g.
+```
+nohup ./kubelet \
+  --kubeconfig=/root/kubeconfig.yaml
+  --v=2 \
+  --address=0.0.0.0 \
+  --enable-server=true \
+  --network-plugin=cni \
+  --cni-conf-dir="/etc/cni/net.d" \
+  --cni-bin-dir="/opt/cni/bin/" \
```

Comment:
@dcbw PTAL
This example should be in sync with whichever directory we choose for the rpm installation. Likely /usr/libexec/cni ?

/cc @pecameron